### DISCUSSION TITLE
Add CIS for Azure Kubernetes Service (AKS) to kube-bench

### DISCUSSION THREAD

**Discussion by miwithro** [aquasecurity/kube-bench#882] [Development]
AKS CIS Benchmark should be the default to kube-bench executions on AKS.

![image](https://user-images.githubusercontent.com/50749048/119887846-a7bdb680-bf02-11eb-8e5a-3879f0e8ca25.png)

**Comment by tspearconquest** [comment]
These are already present. They are in cfg/aks-1.0 in the main branch.
They do need to be updated though as a new version of the CIS benchmarks for AKS was released on June 3.
